package.json

```diff
@@ -5,13 +5,10 @@
   "main": "index.js",
   "scripts": {
     "pretest": "phplint src/*.php src/**/*.php src/**/**/*.php",
-    "manual-deploy-development": "./node_modules/.bin/node-lambda deploy -e development -f config/var_development.env -S config/event_sources_development.json -b subnet-f4fe56af -g sg-1d544067 -P nypl-sandbox -o arn:aws:iam::224280085904:role/lambda_basic_execution --layers arn:aws:lambda:us-east-1:946183545209:layer:lib-for-node10-wrapped-php7:1",
     "manual-deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f config/var_qa.env -S config/event_sources_qa.json -b subnet-f35de0a9,subnet-21a3b244 -g sg-aa74f1db -o arn:aws:iam::946183545209:role/lambda-full-access -P nypl-digital-dev --layers arn:aws:lambda:us-east-1:946183545209:layer:lib-for-node10-wrapped-php7:1",
```

Comment:
whats the difference between manual and not manual deploy?

Reply:
Manual deploy uses a named profile (not available to travis). Not-manual deploy uses a key and secret (available to travis).